websites/F/Facebook/dist/metadata.json

```
@@ -22,7 +22,7 @@
 		"vi_VN": "Kết nối với bạn bè, người thân và những người khác bạn biết. Chia sẻ hình ảnh và video, gửi tin nhắn và nhận các thông báo."
 	},
 	"url": "www.facebook.com",
-	"version": "3.2.12",
+	"version": "3.2.13",
```

It seems like functionality for profiles didn't exist prior to this. If so, you are adding a feature so this should be a minor bump
```suggestion
	"version": "3.3.0",
```

---

profile <shrishidahiya121>